ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/location/keyword.js

```diff
@@ -101,58 +98,50 @@ class Keyword extends React.Component {
   }
   render() {
     const suggester = this.props.suggester || (input => this.suggester(input))
-    const renderLoadingDisplay =
-      this.props.renderLoadingDisplay !== undefined
-        ? this.props.renderLoadingDisplay
-        : true
     const {
       polygon,
-      cursor,
+      setState,
+      setBufferState,
       polygonBufferWidth,
       polygonBufferUnits,
       polyType,
-      loadingMessage,
-      minimumInputLength,
-      placeholder,
     } = this.props
     const { value, loading, error } = this.state
     return (
       <div>
         <AutoComplete
           value={value}
           onChange={option => this.onChange(option)}
-          minimumInputLength={minimumInputLength || 2}
-          placeholder={placeholder || 'Pan to a city, country, or coordinate'}
+          minimumInputLength={2}
+          placeholder="Pan to a region, country, or coordinate"
```

Comment:
❓ How important is it to remove these props? We're using Keyword downstream for a location search without a geo feature. We're using renderLoadingDisplay (to get rid of the unrelated loading message) and the placeholder (because we're not panning). 

Perhaps we can spread the autocomplete props `{...autocompleteProps}` and make the loading message more generic?

![image](https://user-images.githubusercontent.com/22667297/76321007-af158780-629e-11ea-9290-41e75d2bb811.png)

Reply:
I can revert this specific change - in the 2.19.x `keyword.js`, this change doesn't exist. Looks like I just took the older version of it.

Reply:
That works for me. Thanks!